cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/testbeds/ExperimentRunner.java

```diff
@@ -576,11 +629,59 @@ public boolean isVerbose() {
      * affect the verbosity of individual experiments executed. Each
      * {@link SimulationExperiment} has its own verbose attribute.
      *
-     * @param verbose true if the results have to be output, falser otherwise
+     * @param verbose true if the metricsMap have to be output, falser otherwise
```

Comment:
change the javadoc of the verbose parameter to:
true if results have to be output, false otherwise